mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
-  - **Технически:** FastAPI обрабатывает POST-запрос с данными дневника и сохраняет их в базе PostgreSQL.
+4. **Социальные взаимодействия**
+   - Обмен дневниками через ссылки.
+   - Лента активности с обновлениями друзей.
 
-### 2. Добавление мультимедиа
-- **Как пользователь**, я хочу загружать фотографии и видео, чтобы мои записи выглядели живыми и визуально привлекательными.
-  - **Контекст:** После создания дневника пользователь добавляет медиафайлы и просматривает их предварительно.
-  - **Технически:** Загружаемые файлы отправляются на сервер через FastAPI и сохраняются в облачном хранилище (например, AWS S3).
+5. **Приватность и настройки**
+   - Настройка видимости записей: публичные, по ссылке, личные.
 
-### 3. Организация записей
-- **Как пользователь**, я хочу упорядочивать свои записи по дням и местам, чтобы видеть путешествие в хронологическом порядке.
-  - **Контекст:** Пользователь добавляет новые записи с указанием дат и местоположений.
-  - **Технически:** В базе данных каждая запись связывается с дневником через внешние ключи.
+6. **Геймификация**
+   - Достижения и награды за активность.
+   - Баллы за посещение новых мест.
 
-### 4. Автоматическое отслеживание маршрутов
-- **Как пользователь**, я хочу, чтобы приложение автоматически записывало мой маршрут с использованием GPS, чтобы избежать ручного ввода.
-  - **Контекст:** Пользователь включает GPS, и приложение фиксирует передвижения в реальном времени.
-  - **Технически:** FastAPI принимает данные о местоположении через WebSocket или периодические POST-запросы.
+7. **Премиум-функции**
+   - Расширенное хранилище.
+   - Уникальные шаблоны оформления.
+   - Генерация видеоисторий.
 
 ---
 
-## Категория 2: Социальные взаимодействия
+## **Технологии**
+- **Frontend**: ---
+- **Backend**: FastAPI  
+- **База данных**: PostgreSQL  
+- **Облачное хранилище**: AWS S3 (или аналог)  
+- **API для карт**: Google Maps API или Mapbox  
 
-### 5. Обмен дневниками
-- **Как пользователь**, я хочу делиться своими дневниками с друзьями через уникальную ссылку, чтобы они могли увидеть мои путешествия.
-  - **Контекст:** Пользователь нажимает "Поделиться", и приложение генерирует уникальную ссылку.
-  - **Технически:** API создаёт временный токен доступа для предоставления защищённого просмотра дневника.
+---
 
-### 6. Лента активности
-- **Как пользователь**, я хочу видеть обновления друзей в своей ленте, чтобы оставаться в курсе их путешествий.
-  - **Контекст:** Пользователь открывает ленту активности и видит новые записи или дневники друзей.
-  - **Технически:** FastAPI возвращает список обновлений с использованием пагинации для оптимизации.
+## **User Stories**
 
----
+### **Категория 0: Авторизация и регистрация**
+1. **Регистрация нового пользователя**  
+   _Как новый пользователь, я хочу зарегистрироваться, чтобы получить доступ ко всем функциям платформы._  
+   - **Контекст**: Заполнение email, имени, пароля; подтверждение через email.  

```

Comment:
Here need to know how you are sending email, with what service - checkout Sendgrid